<launch>

	<node pkg="velma_common" name="cleaning" type="reset_shm_comm" />

	<node pkg="velma_task_cs_ros_interface" name="initialization" type="initialize_robot.py" />

	<include file="$(find velma_ros_plugin)/launch/velma_planner.launch" />

	<include file="$(find velma_common)/launch/octomap_server.launch" />

	<node name="world_torso_tf_publisher" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 world torso_base 100" />

	<include file="$(find velmobil_navigation)/launch/laserscan_multi_merge.launch"/>

	<arg name="nodelet_manager_name" value="nodelet_manager"/>
	<arg name="publish_nav_vel_topic" value="/mux_vel_nav/cmd_vel"/>
 	
	<node pkg="nodelet" type="nodelet" name="$(arg nodelet_manager_name)" args="manager" />
	<node pkg="nodelet" type="nodelet" name="yocs_cmd_vel_mux" args="load yocs_cmd_vel_mux/CmdVelMuxNodelet $(arg nodelet_manager_name)" output="screen">
    	<param name="yaml_cfg_file" value="$(find building_velma)/src/param/cmd_mux_config.yaml"/>
	</node>

	<node pkg="rviz" name="rviz" type="rviz" />

</launch>